### DISCUSSION TITLE
how to use mutmap+?

### DISCUSSION THREAD

**Discussion by highlei** [YuSugihara/MutMap#8] [Q&A]
mutmap and QTL-seq are the best tools for BSA.
How to use mutmap+?

**Comment by xukaili** [comment]
https://genome-e.ibrc.or.jp/resource/mutmap

**Comment by xukaili** [comment]
> mutmap and QTL-seq are the best tools for BSA.
> How to use mutmap+?
try https://genome-e.ibrc.or.jp/resource/mutmap
